mailbox/api/src/main/java/org/apache/james/mailbox/FlagsBuilder.java

```diff
@@ -22,6 +22,7 @@
 import javax.mail.Flags;
 
 public class FlagsBuilder {
+    public static final Flags.Flag[] ALL_APPLICABLE_FLAGS = {Flags.Flag.ANSWERED, Flags.Flag.DELETED, Flags.Flag.DRAFT, Flags.Flag.SEEN, Flags.Flag.FLAGGED };
```

Comment:
I believe this array of flags does not belong here.

Maybe put it in an interface dedicated to it. Maybe put it in MessageMapper / MessageManager APIs...